### PR TITLE
Add cpio

### PR DESCRIPTION
**Pull request by sogaiu**

### CODE REVIEW

File: community/cpio.hpkg [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+               (join-pkg-paths ":" "/bin"
+                               [core/awk
+                                core/coreutils
+                                core/gcc
+                                core/grep
+                                core/make
+                                core/sed]))
+    (os/setenv "CFLAGS" *default-cflags*)
+    (os/setenv "LDFLAGS" *default-ldflags*)
+    #
```
Comment (andrewchambers) [inline community/cpio.hpkg:23]: Are these empty comments intentional?

Comment (sogaiu) [reply community/cpio.hpkg:23]: Yes, I find they improve readability and I don't like whitespace.

Comment (andrewchambers) [reply community/cpio.hpkg:23]: Hmm not sure how I feel about that. How badly does white space annoy you? :P

Comment (sogaiu) [reply community/cpio.hpkg:23]: Ha ha ha.

So when you saw the blank comment, I imagine you wondered what it was doing there.

I have that type of reaction to whitespace in a function or macro body in lisps.  I wonder why there is space.

If it is a problem to have a blank comment, I can insert text afterwards.

**Pull request merged**